scripts/gcloud_adapter.py

```diff
@@ -45,6 +83,57 @@ def update_indexes(index_yaml_path, app_name):
         index_yaml_path, '--project=%s' % app_name])
 
 
+def get_indexes(app_name):
+    """Obtains indexes serving on the server.
+
+    Args:
+        app_name: str. The name of the GCloud project.
+
+    Returns:
+        list. A list of dict of serving indexes.
```

Comment:
Are the indexes returned guaranteed to be "serving"? If not, please rephrase this line and the first line of the docstring.

Reply:
Done.